test/index.test.ts

```diff
@@ -77,8 +82,8 @@ describe("Testing Toucan-SDK", function () {
 
       await toucan.redeemAuto("NCT", parseEther("1.0"));
 
-      expect(await nct.balanceOf(addr1.address)).to.be.eql(
-        nctBalanceBefore.sub(parseEther("1.0"))
+      expect(formatEther(await nct.balanceOf(addr1.address))).to.be.eql(
```

Comment:
is that the right approach to format all balances to string and compare those? doesn't `balanceOf` already return the amount in wei as `BigNumber` and `parseEther` also returns `BigNumber` and at this point you can check equality on both `BigNumber`'s?

Reply:
Good point @lazaralex98, hex in error logs is annoying. Cool workaround!

Reply:
yeah, good point 👍